README.md

```diff
@@ -14,15 +15,23 @@ There are three npm packages for easier use:
 
 ## Documentation
```

Comment:
I'd change the order to put `What is Airship?` first, and then the documentation link.